Simplify get started index (#1434)
iterative/dvc.org: content/docs/tutorials/get-started/index.md

# Get Started with DVC!
# Get Started

DVC is a **data version control**, pipeline management, and experiment
management tool that brings existing engineering toolsets and practices to data
science and machine learning. In this guide we will introduce the basic features

> and concepts of DVC step by step.

Reviewer: So no short intro after all? There's really no other better place to introduce what DVC is. Unless we review/update the home page (https://dvc.org/) — which needs some updating per 1.0 anyway.

Author: > which needs some updating per 1.0 anyway

1.0 does not directly affect it I would say

> So no short intro after all? 

I'm simplifying everything for now. Also, I just don't think this single paragraph is very useful to be honest. I won't help people understand it better. We'll figure out eventually how to deal with it.

Reviewer: OK. I guess /doc is a better place anyway.

> 1.0 does not directly affect it I would say (home page)

`dvc run` needs -n for staters 😬 

And scrolling down, maybe the features could match the bullets in this new Get Stared.

Author: > dvc run needs -n for staters

it's fine to keep at is in the animation, also we'll be redoing it completely (not related to 1.0)

> And scrolling down, maybe the features could match the bullets in this new Get Stared.

potentially yes, it can benefit from mentioning params, plots. But the way landing page is done now it's a minor change. High level structure is not affected.

### ⚙️ Expand to prepare the project

In expandable sections that start with the ⚙️ emoji, we'll be providing more
information for those trying to run the commands. It's up to you to pick the
best way to read the material - read the text (skip sections like this, and it
should be enough to understand the idea of DVC), or try to run them and get the
fist hand experience.

We'll be building an NLP project from scratch together. The end result is
published on [Github](https://github.com/iterative/example-get-started).

Let's start with `git init`:

```dvc
$ mkdir example-get-started
$ cd example-get-started
$ git init
```

</details>

```dvc
$ dvc init
```

A few [directories and files](<(/doc/user-guide/dvc-files-and-directories)>) are
created that should be added into Git:

```dvc
$ git status
Changes to be committed:
        new file:   .dvc/.gitignore
        new file:   .dvc/config
        ...
$ git commit -m "Initialize DVC"
```

DVC functionality can be split into layers and we'll explore them one by one in
the next few sections:

- [**Data management**](/doc/tutorials/get-started/data-pipelines) is the core

>   part of DVC for large files, datasets, ML models versioning and efficient
>   sharing. We'll show how to use a regular Git workflow, without storing large
>   files with Git. Think "Git for data".

Reviewer: Copy edits:
```suggestion
  feature of DVC for large files, datasets, ML model versioning and
  sharing. We'll cover how to use a regular Git workflow without storing large
  files with Git. Think "Git for data".
```
> Needs wrapping.

Reviewer: Maybe "efficient" -> "easy", "unlimited", or some other term, depending what you wanted to convey here.

Reviewer: Hmmm "efficient sharing" sounds very low-level to me. Do you mean the transfer of files uploaded/downloaded is properly implemented?

Author: No, it is low level. And I wanted to convey that it is optimized in certain ways.

Let's not debate this. The whole get started will be redone anyway right after the release. The goal here is to simplify it a bit, get it back to the NLP project and sync with 1.0. So, let's not even spend much time on terminology. It's not an attempt to make a good next iteration of get started.

- [**Data pipelines**](/doc/tutorials/get-started/data-pipelines) desribe how

>   model or some data artifact are built with an efficient way to reproduce them.

Reviewer: ```suggestion
  models and other data artifacts are built, and provide an efficient way to reproduce them.
```

>   Think "Makefiles done right" for data and ML projects.

Reviewer: Maybe just
```suggestion
  Think "Makefiles for data and ML".
```

> - [**Experiments**](/doc/tutorials/get-started/experiments) attach parameters,
>   metrics, plots. You can capture and navigate experiments not leaving Git.
>   Think "Git for ML".

Reviewer: ```suggestion
- [**Experiments**](/doc/tutorials/get-started/experiments) attach parameters,
  metrics, and plots. You can capture and navigate experiments without leaving Git.
  Think "Git for ML".
```
> Needs rewrap.

But I don't get this last part about not leaving Git or "Git for ML". Sounds redundant with bullet one's "Git for data". Maybe this bullet doesn't need the last sentence.
